tests/framework/utils/k8s_helper.go

```diff
@@ -321,6 +321,7 @@ func (k8sh *K8sHelper) ResourceOperationFromTemplate(action string, podDefinitio
 //ResourceOperation performs a kubectl action on a pod definition
 func (k8sh *K8sHelper) ResourceOperation(action string, manifest string) (string, error) {
 	args := []string{action, "-f", "-"}
+	logger.Infof("kubectl create manifest:\n%s", manifest)
```

Comment:
how verbose will this be in the test output? but if it's helpful for debugging, makes sense to add it even if a bit verbose.

Reply:
It prints out the whole yaml created, so it makes it a fair bit more verbose near the beginning of tests. It was helpful to me when I was trying to mentally map the end result back to what would have created it, but it's certainly not an always-needed kind of thing. I was on the fence about it because it adds several large blocks of yaml to the output. I opted into keeping it thinking that the output is only really inspected when things are going wrong, at which point seeing the yaml 'inputs' are either helpful or can be fairly quickly skipped by human eye if unhelpful based on my experience. Being still a little on the fence, I'm happy to remove it, but it does seem useful enough that it might be worth the extra text.

Reply:
Let's go ahead and add it. There's a fine line between useful and cluttered with logging. We just might turn it off again if it is too much.  :)